packages/usa-in-page-navigation/src/styles/_usa-in-page-navigation.scss

```diff
@@ -0,0 +1,121 @@
+@use "uswds-core" as *;
+@use "sass:color";
+
+.usa-in-page-nav-container {
+  align-items: flex-start;
+  display: flex;
+
+  // Set main landmark max width to ensure in-page nav will fit
+  main,
+  [class^="main"] {
```

Comment:
Do we need to support the `[class^="main"]` as well?

Reply:
The thinking here was that some sites might have a div with the class "main-content". I can remove if you think just main would cover it?